### DISCUSSION TITLE
Why does the PRO label appear in my GitHub Profile when I seem to be on the free plan?

### DISCUSSION THREAD

**Discussion by rhit-parsonjc** [community/community#127152] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

As you can see on my profile (https://github.com/rhit-parsonjc), I have the PRO label. However, as shown on my Billing Summary page (https://github.com/settings/billing/summary), I am currently on the GitHub Free plan and have 2000 minutes of Actions.
![GitHub Plan](https://github.com/community/community/assets/43769180/844f9a42-82f6-4b9f-aa53-6f7cce9972bc)

**Comment by Adnan25z** [comment]
If you are a student, the GitHub Student Developer Pack is likely the reason for the discrepancy you're seeing. The PRO label on your profile indicates that you have access to certain PRO features through the Student Pack, while your billing information correctly shows you as being on the Free plan. This setup is normal for students using the Student Developer Pack.

**Reply by rhit-parsonjc** [reply]
Which features are included in the Student Developer Pack that are not in the free plan? It doesn't seem, for example, that I have access to GitHub Copilot, even though it seems to be included in student plans (https://github.com/edu/students).
![GitHub Copilot](https://github.com/community/community/assets/43769180/d152866c-318f-44d9-a311-fff8b69c83b5)

**Comment by Adnan25z** [comment]
https://github.com/github-copilot/free_signup

try this link

**Reply by rhit-parsonjc** [reply]
I thought that if I was on the student plan, I would automatically have free access to GitHub Copilot and other Pro features, but this doesn't seem to be the case. Is there a place I can verify whether or not I have access to the Student Developer Pack features?

**Reply by ebndev** [reply]
Hi @rhit-parsonjc! You can post / view discussions in our [GitHub Education](https://github.com/orgs/community/discussions/categories/github-education) category about verification if you're still having this issue.

**Comment by BclayDrius** [comment]
I thought that someone said that I was a pro :(
